test/concurrency/testing_transaction_util.cpp

```diff
@@ -212,6 +214,50 @@ storage::DataTable *TestingTransactionUtil::CreateTable(
   return table;
 }
 
+storage::DataTable *TestingTransactionUtil::CreateTableWithoutIndex(
+    std::string database_name, std::string schema_name, std::string table_name) {
+  LOG_INFO("database name = %s", database_name.c_str());
+
+  auto id_column = catalog::Column(
+      type::TypeId::INTEGER, type::Type::GetTypeSize(type::TypeId::INTEGER),
+      "id", true);
+  auto value_column = catalog::Column(
+      type::TypeId::INTEGER, type::Type::GetTypeSize(type::TypeId::INTEGER),
+      "value", true);
+
+  std::unique_ptr<catalog::Schema> table_schema(
+      new catalog::Schema({id_column, value_column}));
+
+  LOG_INFO("schema name = %s", schema_name.c_str());
+  auto catalog = catalog::Catalog::GetInstance();
+  // Create Database and Table
+  auto &txn_manager = concurrency::TransactionManagerFactory::GetInstance();
+  auto txn = txn_manager.BeginTransaction();
+  LOG_INFO("table name = %s", table_name.c_str());
+  catalog->CreateDatabaseWithoutIndex(database_name, txn);
+  txn_manager.CommitTransaction(txn);
+  LOG_INFO("create database %s", database_name.c_str());
+
+  txn = txn_manager.BeginTransaction();
+  catalog->CreateSchema(database_name, schema_name, txn);
+  txn_manager.CommitTransaction(txn);
+  LOG_INFO("create schema %s", schema_name.c_str());
+
+  txn = txn_manager.BeginTransaction();
+  catalog->CreateTable(database_name, schema_name, table_name, std::move(table_schema), txn);
```

Comment:
Should call CreateTableWithoutIndex() ?

Reply:
No. It is just CreateTable. Although CreateTable creates primary index for that table, we want to build other index explicitly for the table and test insert, update, and delete on that index. There is no CreateTableWithoutIndex in the catalog.cpp